src/core/server/saved_objects/routes/export.ts

```diff
@@ -57,12 +57,20 @@ export const registerExportRoute = (router: IRouter, config: SavedObjectConfig)
           search: schema.maybe(schema.string()),
           includeReferencesDeep: schema.boolean({ defaultValue: false }),
           excludeExportDetails: schema.boolean({ defaultValue: false }),
+          workspaces: schema.maybe(schema.arrayOf(schema.string())),
```

Comment:
Could we reuse `workspacesValidator` in utils?

Reply:
The workspaces is different when in query and in body,in workspaces we only accept array of string while in query user may only pass one workspace to backend and we should do some format to transfer the single string to be a string[].